src/platforms/common/data-management/debug-files/source-context.mdx

```diff
@@ -17,3 +17,10 @@ Source bundles show up as regular debug files on the _Debug Files_ settings
 page. They are designated as "source bundle" and feature a `sources` tag. To
 match them with crash reports, they carry the same debug file as the respective
 debug information file they have been created from.
+
+<PlatformSection supported={["dotnet"]}>
+
+Alternatively, you can build your project with `<EmbedAllSources>true</EmbedAllSources>` project property to produce
+a Portable PDB with embedded source code. You can upload these PDB files using `sentry-cli` as usual.
```

Comment:
Ideally we point the users to the documentation of .NET that covers _Sentry CLI via MSBuild_ done here: https://github.com/getsentry/sentry-docs/pull/6140/

I believe we should rename "CLI Integration" to "Sentry CLI via MSBuild" since "integration" is such a bloated term. cc @mattjohnsonpint 

```suggestion
a Portable PDB with embedded source code. 
```